web/docs/e2e-tests.md

```diff
@@ -0,0 +1,46 @@
+# End-to-End Tests (Web)
```

Comment:
I think we should add a short section here about how to run E2E tests locally.

Reply:
Something differen from `### Local testing` below? Please add more infos. IS it to texty too short, or should I just include docker as a solution?